tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -8404,6 +8433,7 @@ declare namespace ts {
     }
     type FileWatcherCallback = (fileName: string, eventKind: FileWatcherEventKind, modifiedTime?: Date) => void;
     type DirectoryWatcherCallback = (fileName: string) => void;
+    type BufferEncoding = "ascii" | "utf8" | "utf-8" | "utf16le" | "ucs2" | "ucs-2" | "base64" | "latin1" | "binary" | "hex";
```

Comment:
Requoting @DanielRosenwasser:

> Kind of wish this didn't need to be exposed. Especially at the top level.

And me: 

> I could undo this bit, but that would unfortunately force everyone to cast the Node function to one in our existing API which accepts any string. Of course, that's feasibly what external users have to do now, too. We just don't experience it in our repo thanks to the lack of SFT.

(Thread got broken thanks to the module merge.)